Formula/garden-cli.rb

```diff
@@ -1,7 +1,7 @@
 class GardenCli < Formula
   desc "Development engine for Kubernetes"
   homepage "https://garden.io"
-  url "https://github.com/garden-io/garden/releases/download/0.12.46/garden-0.12.46-macos-amd64.tar.gz"
+  url "https://download.garden.io/core/0.12.46/garden-0.12.46-macos-amd64.tar.gz"
```

Comment:
@10ko afaik this file is automatically generated by the release GH action. I used the deprecated action in the last 2 releases. We might need to inspect the deprecated GH release action config to ensure the right URL in the download link. It's being injected over the `url "{{{tarballUrl}}}"` in https://github.com/garden-io/garden/blob/main/support/homebrew-formula.rb#L4.

I haven't found a way how the `tarballUrl` was defined.

Reply:
Looks like it's here: https://github.com/garden-io/update-homebrew-action/blob/8f5f7e8b9ab33e0ed39851013a9187c2154fd69b/src/index.ts#L64

I can send a PR for that instead!

Reply:
Done! https://github.com/garden-io/update-homebrew-action/pull/32

Reply:
Thanks a lot, @aviaviavi! Sorry for the late reply. We'll review that PR soon.